MergeSharpWebAPI/Program.cs

```diff
@@ -93,26 +97,46 @@ private static async Task Main(string[] args)
             var serializedLwwSet = JsonConvert.SerializeObject(myLWWSetService.Get(1));
 
             var requestData = new StringContent(serializedLwwSet, Encoding.UTF8, "application/json");
-            //string myContent = await requestData.ReadAsStringAsync();
-            //Console.WriteLine("im porinting data");
-            //Console.WriteLine(myContent);
 
             Console.WriteLine("Sending Put Request for front-end");
             var result = await client.PutAsync(
                 "https://localhost:7009/LWWSet/SendLWWSetToFrontEnd", requestData);
             Console.WriteLine(result);
```

Comment:
can we put a `TODO: check this result for errors` comment? may be helpful catching bugs in future